pkg/iscsi/v1alpha1/iscsi_util.go

```diff
@@ -467,9 +467,8 @@ func (util *ISCSIUtil) DetachDisk(
 		return err
 	}
 
-	if pathExists, pathErr := mount.PathExists(targetPath); pathErr != nil {
-		return fmt.Errorf("Error checking if path exists: %v", pathErr)
-	} else if !pathExists {
+	pathExists, pathErr := mount.PathExists(targetPath)
```

Comment:
we should be having `pathErr != nil` error check as well ?

Reply:
That is not required, when you check inside the mount.PathExists function all the cases are handled